storage/ndb/src/ndbapi/NdbOperation.cpp

```diff
@@ -164,6 +164,7 @@ int NdbOperation::init(const NdbTableImpl *tab, NdbTransaction *myConnection) {
   theInterpretIndicator = 0;
   theInterpretInsertIndicator = 0;
   theDistrKeyIndicator_ = 0;
+  theTTLPurgeWindowSize_ = 0;
```

Comment:
Gather the extra _ in the name can be removed, don't think it serves any purpose